packages/api/db/migration/20230920151515_add_guernsey_aland_to_countries.js

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022, 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+export const up = async function (knex) {
+  const GuernseyExists = await knex('countries')
+    .select('*')
+    .where('country_name', 'Guernsey')
+    .first();
+  const AlandExists = await knex('countries').select('*').where('country_name', 'Guernsey').first();
```

Comment:
'Guernsey' => "Åland Islands"...?

Reply:
Wooooops gotta make at least one mistake per PR haha thanks!

Reply:
It was also meant to be country_code instead of name since its more unique -- sorry!

Reply:
Didn't notice that even at all!